### DISCUSSION TITLE
Trusted cluster

### DISCUSSION THREAD

**Discussion by PallaviPalisetti** [gravitational/teleport#11811] [Q&A]
Hi there,
I have teleport installed on a virtual server, which is the root cluster, and a leaf cluster teleport-cluster helm chart is installed on the K8s. Using teleport-cluster (leaf cluster), we can access applications and k8s.

The leaf cluster is connected to the root cluster and is able to access the leaf cluster applications from the root cluster. But, there is a problem accessing the k8s in the leaf cluster from the root cluster.

**Comment by webvictim** [comment]
What's the problem? What error do you get when you try?

**Reply by PallaviPalisetti** [reply]
from teleport-cluster login I have no problem to access the k8s(staging) from leaf cluster login. when I try to access the k8s(staging) form root cluster login by switching to leaf cluster when I try run (kubectl get pods) throws error no resources found.

**Reply by webvictim** [reply]
Please share the output of these commands. Replace:
- `root.example.com` with your root cluster proxy
- `leaf.example.com` with the name of your leaf cluster from `tsh clusters`
- `kube-cluster` with the name of your leaf Kubernetes cluster from `tsh kube ls`

`tsh login --proxy=root.example.com`
`tsh clusters`
`tsh login leaf.example.com`
`tsh kube ls`
`tsh kube login kube-cluster`
`kubectl get pods`
